### DISCUSSION TITLE
Azure open ai , assistant api , not useing the code interpreter

### DISCUSSION THREAD

**Discussion by Riser01** [danny-avila/LibreChat#4932] [Troubleshooting]
### What happened?

I have configured the chat to use azure open ai , assistant , 
but it cant use the code interperter from the chat , app but its working from the azure openai's default ui

eg prompt 
"i am working on a demo , can u create a demo tabel and fill it woth dummy data , for the followinf columns  
&nbsp;  
email , UDS_Status , Country ,Company name 
&nbsp;  
give me the file to download"

### Steps to Reproduce

eg prompt 
"i am working on a demo , can u create a demo tabel and fill it woth dummy data , for the followinf columns  
&nbsp;  
email , UDS_Status , Country ,Company name 
&nbsp;  
give me the file to download"

any idea why it is different for the same request between the azure openai ui and the chat app 

<img width="1745" alt="image" src="https://github.com/user-attachments/assets/e0e5b031-5c3c-4a0d-8df6-bb2e86f39633">
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/d06d4797-0b46-4e79-9af0-b5e9edebf19b">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxesse** [comment]
Check what api version you're using for azure assistants in librechat.yaml, librechat only supports v1 assistants, it never got v2 support and i think at this point it's unlikely to come, you should use 2024-02-15-preview as it's the last api version with v1 assistant support. Other versions will cause varying levels of weird behaviour, from what you saw now, to downright erroring out.
